bliss._.js

```diff
@@ -58,10 +58,12 @@ Object.defineProperty(Array.prototype, _, {
 if (self.EventTarget && "addEventListener" in EventTarget.prototype) {
 	var addEventListener = EventTarget.prototype.addEventListener,
 	    removeEventListener = EventTarget.prototype.removeEventListener,
-	    equal = function(callback, capture, l){
-	    	return l.callback === callback && l.capture == capture;
+	    equal = function(callback, capture, l) {
+			return l.callback === callback && l.capture == capture;
```

Comment:
Please do, I really like my smart tabs and would hate to see them replaced with dumb tabs :)